cpu/native/include/netdev_raw802154.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
```

Comment:
I think the difference to Kaspar's contribution is large enough that you can replace this with your own copyright note ;-).